hob/.htaccess

```diff
@@ -6,6 +6,18 @@ Options +FollowSymLinks
  
 RewriteEngine on
  
+ 
+#Rewrite rules for hdo vocabulary (term dereferencing to docs)
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml|text/\*|\*/\*)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^(HDO_[0-9]+)$ https://saidfathalla.github.io/Helmholtz-Digitisation-Ontology-Documentation/index-en.html#https://purls.helmholtz-metadaten.de/hob/$1  [R=303,NE,L]
```

Comment:
please clone to IAS-9 github and put there (i.e. dont use your personal account)